guides/v2.2/test/testing.md

```diff
@@ -104,16 +104,36 @@ Please see the [System Administrators Guide on Running Tests][] for more informa
 
 Each of the test types listed above corresponds to a subdirectory in `<magento2 root dir>/dev/tests`.
 
-    dev/tests  
-    ├── acceptance      (since v2.2.4)
-    ├── api-functional  
-    ├── functional  
-    ├── integration  
-    ├── js  
-    ├── static  
-    └── unit  
-
-Each one of these test types has different requirements that must be satisfied before they can be executed.  
+```xml
+<repo_root>
+    <dev/tests/>
```

Comment:
Use the `tree` code block to indicate directory structures like this: 
```tree
<magento 2 root>
└── dev/tests 
    ├── acceptance      (since v2.2.4)
    ├── api-functional  
    ├── functional  
    ├── integration  
    ├── js/jasmine
    ├── setup-integration  
    ├── static  
    └── unit
```

Reply:
Ah I didn't know about that one.